### ISSUE TITLE
CMake Deprecation Warnings

### ISSUE THREAD

**Issue by merkys** [openbabel/openbabel#1860]
Running ``cmake`` with CMake v3.11.2 results in the following CMake deprecation warnings being issued:
```
CMake Deprecation Warning at CMakeLists.txt:11 (cmake_policy):
  The OLD behavior for policy CMP0005 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.


CMake Deprecation Warning at CMakeLists.txt:13 (cmake_policy):
  The OLD behavior for policy CMP0042 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
```
The input files for CMake should be updated in order not to use the deprecated features.

**Comment by baoilleach** [comment]
Do you want to try?

**Comment by merkys** [comment]
I have a very little knowledge of CMake, thus I'd better leave this to someone more experienced.

**Comment by JohnScott623** [comment]
This was fixed in 3.0, see #1948

**Comment by merkys** [comment]
@JohnScott623 thanks for the information. I suppose this issue could be closed now.
